corehq/apps/smsforms/tests/test_sql_session.py

```diff
@@ -7,9 +7,17 @@
 from corehq.apps.sms.handlers.form_session import get_single_open_session_or_close_multiple
 from corehq.apps.smsforms.models import SQLXFormsSession, XFORMS_SESSION_TYPES, XFORMS_SESSION_SMS, \
     XFORMS_SESSION_IVR
+from mock import patch
 from six.moves import range
 
 
+class MockObject(object):
```

Comment:
I think `mock.Mock` does pretty much exactly this. 

```
In [1]: import mock

In [2]: mock.Mock
Out[2]: mock.mock.Mock

In [3]: x = mock.Mock(get_id='contact_id')

In [4]: x.get_id
Out[4]: 'contact_id'
```
Or is there some other reason you made your own?

Reply:
updated: https://github.com/dimagi/commcare-hq/pull/19161/commits/ca0c118a20d5ec1df3ca531800705d9e99122549